### PR TITLE
Feature/merge in missing original texts for global translations

### PR DESCRIPTION
**Pull request by it-vegard**

### CODE REVIEW

File: packages/bygger/src/translations/global/utils.ts [hunk 1]
```diff
@@ -33,4 +34,55 @@ const getCurrentOriginalTextList = (currentTranslation: Array<any>): string[] =>
...
+const getGlobalTranslationsWithLanguageAndTag = (
+  allGlobalTranslations: object,
+  languageCode: string,
+  selectedTag: string
+): TranslationResource => {
+  const globalTranslationsResourcesForSelectedLanguage = allGlobalTranslations[languageCode];
+  const translationsResourceWithSelectedLanguageAndTag = globalTranslationsResourcesForSelectedLanguage.find(
+    isTranslationResourceForSelectedTag(selectedTag)
+  ) || { translations: {} };
+  const missingOriginalTexts = Object.keys(allGlobalTranslations)
```
Comment (it-vegard) [inline packages/bygger/src/translations/global/utils.ts:51]: Should add an if around this code to only run it for tag="skjematekster"

File: packages/bygger/src/translations/global/utils.ts [hunk 2]
```diff
@@ -33,4 +34,55 @@ const getCurrentOriginalTextList = (currentTranslation: Array<any>): string[] =>
...
+    .filter((language) => language !== languageCode)
+    .map((language) => allGlobalTranslations[language])
+    .map((translationResourcesWithDifferentTags) =>
+      translationResourcesWithDifferentTags.find(isTranslationResourceForSelectedTag(selectedTag))
+    )
+    .map((translationResource) => (translationResource ? translationResource.translations : {}))
+    .reduce((missingOriginalTexts, translationsResource): string[] => {
+      if (translationsResource) {
```
Comment (it-vegard) [inline packages/bygger/src/translations/global/utils.ts:59]: Check if this can be removed

File: packages/bygger/src/translations/global/utils.ts [hunk 3]
```diff
@@ -33,4 +34,55 @@ const getCurrentOriginalTextList = (currentTranslation: Array<any>): string[] =>
...
+            (key) =>
+              translationsResourceWithSelectedLanguageAndTag.translations[key] === undefined &&
+              missingOriginalTexts.indexOf(key) === -1
+          ),
+        ];
+      } else {
+        return missingOriginalTexts;
+      }
+    }, []);
+  missingOriginalTexts.forEach(
```
Comment (it-vegard) [inline packages/bygger/src/translations/global/utils.ts:72]: Add as last operation on the chain above

**Review by wen7i — APPROVED** [review]
🥇 🎖️ 🏅 🐐 💨

**Pull request merged**